web/src/beta/components/CheckboxField/index.tsx

```diff
@@ -0,0 +1,50 @@
+import Icon from "@reearth/beta/components/Icon";
+import Text from "@reearth/beta/components/Text";
+import { styled, useTheme } from "@reearth/services/theme";
+
+export type Props = {
+  onClick?: (value: boolean) => void;
+  checked?: boolean;
+  label: string;
+};
+
+const CheckBoxField: React.FC<Props> = ({ onClick, checked, label }) => {
+  const theme = useTheme();
+  return (
+    <Field onClick={() => onClick?.(checked !== undefined ? checked : false)}>
```

Comment:
```suggestion
    <Field onClick={() => onClick?.(!checked)}>
```
You have to pass opposite boolean value to the event